README.md

```diff
@@ -40,34 +40,31 @@ If the SFTP service is used, the question arises regarding how the system will k
 
 1. Clone this repository.
 2. Change directories into the `base-local` directory.
-3. Determine whether to store encrypted files on the local file system or in AWS S3 (*hint: for this initial version, choose filesystem*).
-4. Copy the proper files based on the storage selection; for example:
+3. Adjust if needed values in [.env file](./env)
+4. Determine whether to store encrypted files on the local file system or in AWS S3 (*hint: for this initial version, choose filesystem*).
+5. Copy the proper files based on the storage selection; for example:
     ```bash
     cp docker-compose.yaml.fs docker-compose.yaml
-    cp files/etc/config.yaml.fs files/etc/config.yaml
+    cp etc/config.yaml.fs etc/config.yaml
     ```
-5. Edit the IP address hard-coded in the docker-compose.yaml file to match that of your local host. Also review the port numbers and the passwords provided and change if needed.
 6. When ready, run the Docker Compose command in the base-local directory to instantiate the Docker stack:
     ```bash
     sudo docker-compose up -d
     ```
-7. When the containers are built and deployed, the database is empty; import the database schemas:
+7. Wait till the database starts and is ready to connect to using the following command:
     ```bash
-    sudo ./populate_mysql_databases.sh
+    docker-compose logs -f database
     ```
-    If you execute this command fairly soon after the `docker-compose` command, the MySQL service may not have completed starting up. (Startup takes a full two minutes on our test system.) In this case, you should see some "waiting" and error messages until the service is available to receive SQL queries. Once available, the script will create databases, download and import schemas, and create a user with full privileges on the databases.
-
-8. Although the LCP containers were started in step 6, they failed because there was no database configured. Now that the database is available, shut down the services, then bring them back up so the LCP services can log into the database. The order in which the containers is started is important. The database container must be started and operational before the lcpserver, lsdserver, and testfrontend containers are started. Using `docker-compose restart` has not resulted in a good startup order for us; performing separate down and up operations has resulted in a functional order. 
+8. After the database is fully up and ready restart `lcpserver`, `lsdserver`, and `testfrontend`:
```

Comment:
I think some of the note from my step 8 is handy to explain why we're running the compose up command again. Maybe restore the "Although the LCP containers were started in step 6, they failed because there was no database configured. Now that the database is available," bit from mine right before "restart `lcpserver`," from yours?

Reply:
I changed it accordingly